_data/other.yml

```diff
@@ -279,6 +279,16 @@ websites:
       img: promoter.png
       tfa: No
       twitter: Promoter_io
+      
+    - name: Repairshopr
+      url: http://www.repairshopr.com/
```

Comment:
HTTPS only redirects to HTTP on their main site.  All login pages and the actual product use http; however, their homepage and documentation all uses HTTP.  :/